docs/debugging_crashes.md

```diff
@@ -34,6 +34,16 @@ For example, running these commands in an administrative command prompt will set
 
 After you repro the crash, you should find a dump in the specified folder.
 
+You can automate crash dump collection and filing bugs by following these steps:
+1. Download the script at https://aka.ms/RNW/analyze-crash.ps1, for example to `C:\temp`
+2. Open an admin PowerShell. If you haven't enabled running unsigned scripts yet, do that by running: `Set-ExecutionPolicy Unrestricted`
+3. Run the script and pass it the name of your app's exe: `C:\temp\analyze-crash.ps1 -ExeName MyApp -Repo microsoft/microsoft-ui-xaml`
```

Comment:
>MyApp [](http://example.com/codeflow?start=94&length=5)

Just to confirm, this is just "MyApp" not "MyApp.exe"?

Reply:
correct, the script appends .exe to the `LocalDumps` key